modules/nw-autoscaling-ingress-controller.adoc

```diff
@@ -61,25 +61,37 @@ EOF
 ----
 
 . Define a `TriggerAuthentication` object within the `openshift-ingress-operator` namespace by using the service account's token.
-
++
+.. Define the variable `secret` that contains the secret by running the following command:
++
+[source,terminal]
+----
+$ secret=$(oc get secret -n openshift-ingress-operator | grep thanos-token | head -n 1 | awk '{ print $1 }')
+----
++
 .. Create the `TriggerAuthentication` object and pass the value of the `secret` variable to the `TOKEN` parameter:
 +
 [source,terminal]
 ----
-$ oc apply -f - <<EOF
-apiVersion: keda.sh/v1alpha1
-kind: TriggerAuthentication
-metadata:
-  name: keda-trigger-auth-prometheus
-  namespace: openshift-ingress-operator
-spec:
-  secretTargetRef:
-  - parameter: bearerToken
-    name: thanos-token
-    key: token
-  - parameter: ca
-    name: thanos-token
-    key: ca.crt
+$ oc process TOKEN="$secret" -f - <<EOF | oc apply -f -
```

Comment:
Was scratching my head why i couldn't get this to work on 4.15. Turns out, the `namespace: openshift-ingress-operator` below isn't respected when generating with oc process. You must add the `-n openshift-ingress-operator` here, and you can remove the namespace below.

```suggestion
$ oc process TOKEN="$secret" -f - <<EOF | oc apply -n openshift-ingress-operator -f -
```

Reply:
:heavy_check_mark: